src/Tizen.NUI/src/public/Input/InputMethodContext.cs

```diff
@@ -1104,7 +1104,7 @@ public class CallbackData : Disposable
             /// The state if it owns memory
             /// </summary>
             /// <since_tizen> 5 </since_tizen>
-            //ToDo : raise ACR as [Obsolete("This has been deprecated in API9 and will be removed in API11. Please use SwigCMemOwn which is declared in parent class")]
+            [Obsolete("This has been deprecated in API9 and will be removed in API11. Please use SwigCMemOwn which is declared in parent class")]
```

Comment:
```suggestion
            [Obsolete("This has been deprecated in API9 and will be removed in API11. Use SwigCMemOwn that is declared in the parent class.")
```

Reply:
Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.